src/css/components/Breadcrumbs.scss

```diff
@@ -0,0 +1,33 @@
+@import "../config/index.scss";
+
+// * @define default Breadcrumbs
+
+.Nav-breadcrumb {
+  color: black;
+  font-size: 16px;
+}
+
+a {
+  color: black;
+}
```

Comment:
We cant add this style as this would make all anchors black across whatever app pulls this in. We either need to nest this specific style under the breadcrumb class, or find a different way to accomplish this.

Reply:
Done!